src/app/(proper_react)/redesign/(authenticated)/user/dashboard/fix/data-broker-profiles/start-free-scan/page.tsx

```diff
@@ -2,68 +2,50 @@
+import { getOnerepProfileId } from "../../../../../../../../../db/tables/subscribers";
+import { getCountryCode } from "../../../../../../../../functions/server/getCountryCode";
+import { StepDeterminationData as StepDeterminationData } from "../../../../../../../../functions/server/getRelevantGuidedSteps";
+import { getSubscriberBreaches } from "../../../../../../../../functions/server/getUserBreaches";
+import { getSubscriberEmails } from "../../../../../../../../functions/server/getSubscriberEmails";
+import { StartFreeScanView } from "./StartFreeScanView";
 
-import Image from "next/image";
-import ImageCityScape from "./images/city-scape.svg";
-import styles from "../dataBrokerProfiles.module.scss";
-import { useL10n } from "../../../../../../../../hooks/l10n";
-import { Button } from "../../../../../../../../components/server/Button";
+export default async function StartFreeScanPage() {
+  const countryCode = getCountryCode(headers());
+  if (countryCode !== "us") {
+    redirect("/redesign/user/dashboard");
+  }
 
-export default function StartFreeScan() {
-  const l10n = useL10n();
+  const session = await getServerSession(authOptions);
+  if (!session?.user?.subscriber?.id) {
+    return redirect("/");
+  }
 
-  return (
-    <div className={styles.contentWrapper}>
-      <Image className={styles.cityScape} src={ImageCityScape} alt="" />
-      <div className={styles.content}>
-        <h3>
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-start-free-scan-headline"
-          )}
-        </h3>
-        <p>
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-start-free-scan-content-p1",
-            {
-              data_broker_count: parseInt(
-                process.env.NEXT_PUBLIC_ONEREP_DATA_BROKER_COUNT as string,
-                10
-              ),
-            }
-          )}
-        </p>
-        <p>
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-start-free-scan-content-p2"
-          )}
-          <a href="#">
-            {l10n.getString(
-              "fix-flow-data-broker-profiles-start-free-scan-link-learn-more"
-            )}
-          </a>
-        </p>
-      </div>
-      <div className={styles.buttonsWrapper}>
-        <Button
-          variant="primary"
-          href={
-            "/redesign/user/dashboard/fix/data-broker-profiles/view-data-brokers"
-          }
-        >
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-start-free-scan-button-start-scan"
-          )}
-        </Button>
-        <Button
-          variant="secondary"
-          href={"/redesign/user/dashboard/fix/high-risk-data-breaches"}
-        >
-          {l10n.getString(
-            "fix-flow-data-broker-profiles-start-free-scan-button-skip"
-          )}
-        </Button>
-      </div>
-    </div>
-  );
+  const result = await getOnerepProfileId(session.user.subscriber.id);
+  const onerepProfileId = result[0]["onerep_profile_id"];
```

Comment:
1 or 0 (see check on line 32). Not sure why it returns an array rather than a row or null, but a rework of this function is probably out of scope of this PR :)